File: Observam/TodayVC.m
//
//  TodayVC.m
//  Observam
//
//  Created by Mathieu Godart on 11/04/2016.
//  Copyright © 2016 Godart. All rights reserved.
//

#import "TodayVC.h"
#import "AppDelegate.h"

@interface TodayVC_OLD ()
@property (strong, nonatomic) IBOutlet UIImageView *backgroundView;
@property (strong, nonatomic) IBOutlet UITableViewCell *tipCell;
@property (strong, nonatomic) IBOutlet UILabel *tipOfTheDay;
@end



@implementation TodayVC_OLD



- (void)viewDidAppear:(BOOL)animated {
//    [self performSegueWithIdentifier:@"showTakeDrugScene" sender:nil];
}



- (void)viewDidLoad {
    [super viewDidLoad];

    self.tableView.estimatedRowHeight = 44;

    self.tipOfTheDay.text = @"Lavez-vous les mains avant et après chaque fois que vous avez touché les comprimés.";

    self.tableView.backgroundView = self.backgroundView;
}



- (IBAction)relaunchWizard:(id)sender {
//    globalIsWizardCompleted = false
}



- (CGFloat)tableView:(UITableView *)tableView heightForRowAtIndexPath:(NSIndexPath *)indexPath
{
    return UITableViewAutomaticDimension;
}



- (void)tableView:(UITableView *)tableView willDisplayHeaderView:(UIView *)view
       forSection:(NSInteger)section {

    UITableViewHeaderFooterView *header = (UITableViewHeaderFooterView *)view;
    [header.textLabel setTextColor:[AppD globalTintColor]];
}



- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}



@end

File: Observam/TodayVC.swift
//
//  TodayVC.swift
//  Observam
//
//  Created by Mathieu Godart on 08/05/2016.
//  Copyright © 2016 Godart. All rights reserved.
//

import UIKit

class TodayVC: UITableViewController {

    @IBOutlet var tipCell: UITableViewCell!
    @IBOutlet var tipOfTheDay: UILabel!
    @IBOutlet var backgroundView: UIImageView!

    override func viewDidLoad() {
        super.viewDidLoad()

        // Uncomment the following line to preserve selection between presentations
        // self.clearsSelectionOnViewWillAppear = false

        // Uncomment the following line to display an Edit button in the navigation bar for this view controller.
        // self.navigationItem.rightBarButtonItem = self.editButtonItem()

        self.tableView.estimatedRowHeight = 44

        self.tipOfTheDay.text = "Lavez-vous les mains avant et après chaque fois que vous avez touché les comprimés."

        self.tableView.backgroundView = self.backgroundView
    }



    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }



    @IBAction func relaunchWizard(_ sender: AnyObject) {
        globalIsWizardCompleted = false
    }



    @IBAction func readMore(_ sender: AnyObject) {
        self.navigationController?.tabBarController?.selectedIndex = 1
    }
    
    
    
    @IBAction func didFillUpStock(_ sender: AnyObject) {

        let alert = UIAlertController(title: nil,
                                      message: nil,
                                      preferredStyle: .actionSheet)

        alert.addAction(UIAlertAction(title: "J'ai fait le plein",
            style: .default, handler: { (action: UIAlertAction) in
        }))
        alert.addAction(UIAlertAction(title: "Annuler",
            style: .cancel, handler: { (action: UIAlertAction) in
        }))

        alert.view.tintColor = globalTintColor()

        present(alert, animated: true, completion:nil)
    }



    // MARK: - Table view data source



    override func tableView(_ tableView: UITableView, heightForRowAt indexPath: IndexPath) -> CGFloat {
        return UITableViewAutomaticDimension
    }


    override func tableView(_ tableView: UITableView, willDisplayHeaderView view: UIView, forSection section: Int) {

        let header = view as? UITableViewHeaderFooterView
        header?.textLabel?.textColor = globalTintColor()
    }


//    override func numberOfSectionsInTableView(tableView: UITableView) -> Int {
//        // #warning Incomplete implementation, return the number of sections
//        return 0
//    }
//
//    override func tableView(tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
//        // #warning Incomplete implementation, return the number of rows
//        return 0
//    }

    /*
    override func tableView(tableView: UITableView, cellForRowAtIndexPath indexPath: NSIndexPath) -> UITableViewCell {
        let cell = tableView.dequeueReusableCellWithIdentifier("reuseIdentifier", forIndexPath: indexPath)

        // Configure the cell...

        return cell
    }
    */

    /*
    // Override to support conditional editing of the table view.
    override func tableView(tableView: UITableView, canEditRowAtIndexPath indexPath: NSIndexPath) -> Bool {
        // Return false if you do not want the specified item to be editable.
        return true
    }
    */

    /*
    // Override to support editing the table view.
    override func tableView(tableView: UITableView, commitEditingStyle editingStyle: UITableViewCellEditingStyle, forRowAtIndexPath indexPath: NSIndexPath) {
        if editingStyle == .Delete {
            // Delete the row from the data source
            tableView.deleteRowsAtIndexPaths([indexPath], withRowAnimation: .Fade)
        } else if editingStyle == .Insert {
            // Create a new instance of the appropriate class, insert it into the array, and add a new row to the table view
        }    
    }
    */

    /*
    // Override to support rearranging the table view.
    override func tableView(tableView: UITableView, moveRowAtIndexPath fromIndexPath: NSIndexPath, toIndexPath: NSIndexPath) {

    }
    */

    /*
    // Override to support conditional rearranging of the table view.
    override func tableView(tableView: UITableView, canMoveRowAtIndexPath indexPath: NSIndexPath) -> Bool {
        // Return false if you do not want the item to be re-orderable.
        return true
    }
    */

    /*
    // MARK: - Navigation

    // In a storyboard-based application, you will often want to do a little preparation before navigation
    override func prepareForSegue(segue: UIStoryboardSegue, sender: AnyObject?) {
        // Get the new view controller using segue.destinationViewController.
        // Pass the selected object to the new view controller.
    }
    */

}
